### PR TITLE
Hellonoah

### PR DESCRIPTION
**Pull request by n-k-f**
:boom: ABL homework! :boom: 

:baby:

### CODE REVIEW

File: funhomework.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+feel like maybe I’m starting to get the hang of this?
```
Comment (rsliter) [inline funhomework.txt:1]: :tada:

Comment (emmatangerine) [inline funhomework.txt:1]: you're the :bomb: @n-k-f

**Comment by n-k-f** [comment]
fixed i think?

**Comment by rsliter** [comment]
:boom:

**Pull request merged**